class ValidationError(Exception):
    '''Кастомный Exception.
    
    Ожидается что будет какая-то функция-Родитель которая будет запускать
    фукнцию-Верификатор. Если Верификатор решил что данные не прошли проверку -
    он будет подбрасывать родителю этот эксепшен.
    '''

class OperationPermissionError(Exception):
    '''Кастомный Exception.
    
    Ожидается что в этих эксепшенах будут передаваться сообщения о недостатке
    прав доступа к бизнес-процесса.
    '''
